safe/definitions/reports/components.py

```diff
@@ -346,6 +348,49 @@
     }
 }
 
+analysis_provenance_details_component = {
+    'key': 'analysis-provenance-details',
+    'type': jinja2_component_type,
+    'processor': jinja2_renderer,
+    'extractor': analysis_provenance_details_extractor,
+    'output_format': Jinja2ComponentsMetadata.OutputFormat.String,
+    'output_path': 'analysis-provenance-details-output.html',
+    'template': 'standard-template/'
+                'jinja2/'
+                'analysis-provenance-details.html',
+    'extra_args': {
+        'defaults': {
+            'source': tr('source not available'),
+            'reference': tr('reference unspecified'),
+            'aggregation_not_used': tr('not used')
+        },
+        'header': {
+            'analysis_detail': tr('Analysis details')
+        },
+        'provenance_format': {
+            'hazard_header': tr(
+                'Hazard source'),
+            'hazard_format': tr(
+                '{layer_name} - {source} - '),
+
+            'exposure_header': tr(
+                'Exposure source'),
+            'exposure_format': tr(
+                '{layer_name} - {source} - '),
+
+            'aggregation_header': tr(
+                'Aggregation source'),
+            'aggregation_format': tr(
+                '{layer_name} - {source} - '),
+
+            'impact_function_header': tr(
+                'Impact Function'),
+            'impact_function_format': tr(
+                '{impact_function_name}'),
```

Comment:
Yes for 379, I think we don't need that.
For 384, I guess impact function name were already translated.